tests/features/finish_by_hand.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Finish by hand
+
+***TO DO - This is still a skeleton and needs to be completed.
+After I am found to be eligible, if I click the "Finish by hand" button,
+I should be taken to the correct page.
+If I click continue on that page, I should be taken to the final page.
+
+Scenario: Just open the first page
+    Given I start the interview "docassemble.playground42:juvenile-sealing-intro-9.yml#page11"
```

Comment:
This url has "#page11". I don't think it should make a difference because of the way docassemble works, but who knows, so it may be worth removing. It also may need the additional url param of `new_session=1`. I'm trying to find that out now.

Reply:
Looks like we don't need `new_session=1`. Dev says: "In order to run multiple scenarios in the same interview from the beginning, the "step" for "I launch the interview <interview name>" adds &reset=2 to the URL."